pkgs/servers/openafs/1.8/module.nix

```diff
@@ -11,6 +11,8 @@ in stdenv.mkDerivation rec {
   name = "openafs-${version}-${kernel.modDirVersion}";
   inherit version src;
 
+  patches = [ ./linux-4.20.patch ];
```

Comment:
It is preferable to fetch patch from git directly using `fetchpatch`. It seems like we need these:
http://git.openafs.org/?p=openafs.git;a=patch;h=3c454b39d04f4886536267c211171dae30dc0344
http://git.openafs.org/?p=openafs.git;a=patch;h=aa80f892ec39e2984818090a6bb2047430836ee2

And this one wouldn't hurt:
http://git.openafs.org/?p=openafs.git;a=patch;h=10b02075a262dbe802266ea4bcac3936dff5dd23

Reply:
I did not realize that. I just tried it, but the line numbers have changed significantly between OpenAFS 1.8.2 and git versions, so the patches don't apply. My attempt is in the [openafs-broken](https://github.com/artemist/nixpkgs/tree/openafs-broken) branch of my fork